mvcc/watchable_store_test.go

```diff
@@ -26,6 +26,8 @@ import (
 	"github.com/coreos/etcd/lease"
 	"github.com/coreos/etcd/mvcc/backend"
 	"github.com/coreos/etcd/mvcc/mvccpb"
+	"io/ioutil"
```

Comment:
these should be grouped with the standard libraries above instead of with external packages